src/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionsRail.tsx

```diff
@@ -42,16 +43,22 @@ export class ArtistCollectionsRail extends React.Component<
     // noop
   }
 
+  static defaultProps = {
+    includeTopSpacer: true,
+  }
+
   render() {
-    const { collections } = this.props
+    const { collections, includeTopSpacer } = this.props
     if (collections.length > 3) {
       return (
         <Box>
           <Waypoint onEnter={once(this.trackImpression.bind(this))} />
-          <Sans size="3" weight="medium">
-            Browse by iconic collections
+
+          {includeTopSpacer && <Separator my={3} />}
```

Comment:
Doesn't feel great to have to add this here, but we don't want to show the separator/spacer unless content _loads_, and we only know that from within this element (otherwise we'd have to fetch for data in the parent and avoid any performance benefits we'd get from loading client-side).